tastypie_mongoengine/resources.py

```diff
@@ -97,7 +97,8 @@ def __new__(cls, name, bases, attrs):
                     del(new_class.base_fields[field_name])
             if field_name in new_class.declared_fields:
                 continue
-            if len(include_fields) and field_name not in include_fields:
+            # Meta.fields is None by default
+            if include_fields and len(include_fields) and field_name not in include_fields:
```

Comment:
you can just remove the `len(include_fields)`

Reply:
8a7d7fa9b1f78b511dc789e6a4718370b69d46e0 :white_check_mark: